ports/lib3mf/portfile.cmake

```
@@ -0,0 +1,55 @@
+# Manually clone the repository with submodules
+vcpkg_download_distfile(ARCHIVE
+    URLS "https://github.com/3MFConsortium/lib3mf/archive/refs/tags/v2.3.2.zip"
+    FILENAME "lib3mf-2.3.2.zip"
+    SHA512 5bf888c06a2429bfaca45b5ed96b2a7077a84ca3d1c4d114967eca0798189d9cbe7454eb810dcc18ccf31561efca271c4f68e7666f9f2665a8592df530f0d0bb
+)
+
+# Extract it
+vcpkg_extract_source_archive_ex(
+    OUT_SOURCE_PATH SOURCE_PATH
+    ARCHIVE ${ARCHIVE}
+)
+
+# Apply the patch
+vcpkg_apply_patches(
+    SOURCE_PATH ${SOURCE_PATH}
+    PATCHES "lib3mf_vcpkg.patch"
+)
+
+# Only dynamic libraries (Based on Jan's Port)
+vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
```

Just suggestion:

```suggestion
vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)

vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO 3MFConsortium/lib3mf
    REF "v${VERSION}"
    SHA512 875529759aff6128b32291d0719b95a75a72a3b19f98711b12c9b1076727b231668b3b14604e69bb1fec79b4725fd098f030b7e965072e566bfb061f3086279c
    HEAD_REF master
)

vcpkg_apply_patches(
    SOURCE_PATH ${SOURCE_PATH}
    PATCHES "0001-lib3mf-fix-deps.patch"
)
```

---

More like:

```suggestion
vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)

vcpkg_from_github(
    OUT_SOURCE_PATH SOURCE_PATH
    REPO 3MFConsortium/lib3mf
    REF "v${VERSION}"
    SHA512 875529759aff6128b32291d0719b95a75a72a3b19f98711b12c9b1076727b231668b3b14604e69bb1fec79b4725fd098f030b7e965072e566bfb061f3086279c
    HEAD_REF master
    PATCHES
        0001-lib3mf-fix-deps.patch
)
```
(Don't call `vcpkg_apply_patches` separately.)

---

Sorry I was negligent, please follow @dg0yt 's suggestion.